01-introduction.Rmd

```diff
@@ -45,3 +37,38 @@ What to expect in this book:
 - **Chapter \@ref(c10-ambarom-vignette)**: AmericasBarometer Vignette. A vignette on how to analyze data from the AmericasBarometer, a survey of attitudes, evaluations, experiences, and behavior in countries in the Western Hemisphere. This includes how to make choropleth maps with survey estimates.
 
 Most chapters include code to follow along with.  Each chapter with this code will start with a set-up section.  This will include all of the code needed to load packages and datasets to use in the chapter.  We then provide an overview of the topic along with examples on how to use the functions.  Most chapters end with exercises to work through.  Solutions to these exercises can be found online at [XX].
+
+The two main datasets we use throughout the book are the Residential Energy Consumption Survey [RECS -- @recs-2015-micro] and the American National Election Studies [ANES -- @debell].  To ensure that all readers can easily follow along with the examples, we have provided analytic datasets for readers on OSF^[https://osf.io/gzbkn/?view_only=8ca80573293b4e12b7f934a0f742b957]. To load any data used in the book that is not included in existing packages, we have created a helper function `read_osf()`. Here is the code to read in RECS and ANES datasets: 
+
+```{r}
+#| label: intro-setup
+#| error: FALSE
+#| warning: FALSE
+#| message: FALSE
+library(tidyverse)
+library(survey)
+library(srvyr)
+library(osfr)
+source("helper-fun/helper-functions.R")
+
+recs_in <- read_osf("recs_2015.rds")
+anes_in <- read_osf("anes_2020.rds")
+```
+
+RECS is a household study of energy consumption in the U.S. which provides energy consumption and expenditures data. RECS funded by Energy Information Administration and collects information through energy suppliers through in-person, phone, and web interviews. It has been fielded 14 times between 1950 and 2020 and includes questions about appliances, electronics, heating, air conditioning (A/C), temperatures, water heating, lighting, energy bills, respondent demographics, and energy assistance.  Here is an overview of the data we just read in `recs_in`:
+
+```{r}
```

Comment:
Right now this is a static output which works fine for PDF, but wasn't sure if we wanted to add pagination for the HTML output.  Thoughts?  (would impact ANES below as well).

Reply:
Perhaps using summary or glimpse rather than trying to print it?